docs/modules/extensions/pages/processors/block-processor.adoc

```diff
@@ -0,0 +1,135 @@
+= Block Processor
+
+BlockProcessor is a SyntaxProcessor
+
+== Registration method
+
+A BlockProcessor is registered with `block`.
+
+== DSL methods
+
+include::page$processor.adoc[tag=syntaxproc]
+
+`contexts`::
+Sets the contexts allowed for this block.
+The choices are:
+
+[cols=2,opts=headers]
+|===
+| Context
+| (shortest) Delimiters
+
+| `:paragraph`
+| (no explicit delimiters, end delimiter is a blank line)
+
+| `:open`
+| `--`
+
+| `:listing`
+| `----`
+
+| `:literal`
+| `....`
+
+| `:example`
+| `====`
+
+| `:sidebar`
+| `\****`
+
+| `:quote`
+| `\____`
+
+| `:pass`
+| `\++++++`
+
+| `:table`
+| `\|===`, `,===`, `:===`, or `!===`
+
+| `:comment`
+| `////`
+
+| `:fenced_code`
```

Comment:
The fenced_code context is obsolete.

Reply:
Since this isn't documented anywhere I can find, I'm leaving a comment noting obsolescence in place.